# HG changeset patch
# User Ray Donnelly <mingw.android@gmail.com>
# Date 1471185428 -3600
#      Sun Aug 14 15:37:08 2016 +0100
# Node ID 96f4dd23279f67691b74652b1ef59898dc8d9fdb
# Parent  d992e8de5d80dcc7638362eddc967263a78a00b8
Win32: Add __MINGW_USE_ANSI_STDIO=1 for '%zd' in _rinterface.c

diff --git a/setup.py b/setup.py
--- a/setup.py
+++ b/setup.py
@@ -211,6 +211,7 @@
 
     if sys.platform == 'win32':
         define_macros.append(('Win32', 1))
+        define_macros.append(('__USE_MINGW_ANSI_STDIO', 1))
         if "64 bit" in sys.version:
             define_macros.append(('Win64', 1))
             extra_link_args.append('-m64')
